tests/playwright/pages/wp-admin-page.js

```diff
@@ -20,20 +18,60 @@ exports.WpAdminPage = class wpAdminPage {
 		}
 
 		await this.page.waitForSelector( 'text=Log In' );
-		await this.page.fill( 'input[name="log"]', 'admin' );
-		await this.page.waitForTimeout( 500 );
-		await this.page.fill( 'input[name="pwd"]', 'password' );
+		await this.page.fill( 'input[name="log"]', this.config.user.username );
+		await this.page.fill( 'input[name="pwd"]', this.config.user.password );
 		await this.page.click( 'text=Log In' );
 		await this.page.waitForSelector( 'text=Dashboard' );
 	}
 
-	async openNewPage() {
-		if ( ! await this.page.$( 'text=Create New Page' ) ) {
-			await this.gotoDashboard();
+	async waitForPanel() {
+		await this.page.waitForSelector( '#elementor-panel-header-title' );
+	}
+
+	async createNewPage() {
+		try {
+			await this.page.click( 'text=Create New Page', { timeout: 5000 } );
+		} catch ( err ) {
+			console.error( "Click on Elementor 'Create New Page' button failed" );
+			await this.page.waitForSelector( 'text=Dashboard' );
+			await this.page.click( 'text=Pages' );
+
+			await Promise.all( [
+				this.page.waitForNavigation( { url: '/wp-admin/post-new.php?post_type=page' } ),
+				this.page.click( 'div[role="main"] >> text=Add New' ),
+			] );
+
+			await Promise.all( [
+				this.page.waitForNavigation(),
+				this.skipTutorial(),
+				this.page.click( 'text=← Back to WordPress Editor Edit with Elementor' ),
+			] );
 		}
 
-		await this.page.click( 'text=Create New Page' );
-		await this.page.waitForSelector( '#elementor-panel-header-title' );
+		await this.waitForPanel();
+
+		return new EditorPage( this.page, this.testInfo );
+	}
+
+	async useElementorCleanPost() {
+		await this.page.goto( `/wp-admin/post.php?post=${ CLEAN_POST_ID }&action=elementor` );
+
+		await this.waitForPanel();
+
+		const editor = new EditorPage( this.page, this.testInfo );
+
+		await this.page.evaluate( () => $e.run( 'document/elements/empty', { force: true } ) );
```

Comment:
If you want it to be clean. maybe also reset the page settings

Reply:
Fine for now. i won't get too deep, when it will be required, then we will handle it.